menu.kando.Kando.yml

```diff
@@ -0,0 +1,35 @@
+id: menu.kando.Kando
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: AppRun
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.gnome.Shell
+  - --talk-name=org.kde.KWin
+modules:
+  - name: kando
+    buildsystem: simple
+    sources:
+      - type: file
+        dest-filename: kando.AppImage
+        url: https://github.com/kando-menu/kando/releases/download/v1.6.0/Kando-1.6.0-x86_64.AppImage
+        sha256: 4448419ab790e2623e590439b8b4202d14030e5ee1a4cd2add4286dff329e670
+      - type: file
+        path: menu.kando.Kando.svg
+      - type: file
+        path: menu.kando.Kando.desktop
+      - type: file
+        path: menu.kando.Kando.metainfo.xml
```

Comment:
these should be inside the appimage. I hope you saw the note in https://docs.flathub.org/docs/for-app-authors/requirements#required-metadata

Reply:
They are currently not. At least not in a format suitable for the flatpak. Instead, I have now put them into the upstream repository.

Reply:
What do you mean by " not in a format suitable for the flatpak"

Reply:
`menu.kando.Kando.metainfo.xml` is not included, the icon is only a png, the desktop file I could use. But if I copy the other two files directly from the repository, I can also copy that one. It really does not make a difference, it's the same file after all.

Reply:
It's better to use the ones in appimage and modify whatever keys you need using desktop-file-edit.

Desktop files can take in translations too.

And the metainfo should be included in the upstream repo.

Reply:
Alright, I'll do this in the next release of Kando. I already updated the scripts which generate the Appimage upstream, so with the next version of Kando will be able to improve this part. But for now, I hope it's fine if I include them directly from upstream.